### ISSUE TITLE
Facet Navigation and Screen Reader(s)

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/cho#712]
Putting a ref to https://github.com/projectblacklight/blacklight/issues/1905 here as a reminder to fix this issue upstream.
